Fabric.Identity.API/scripts/Install-Identity-Windows.ps1

```diff
@@ -327,7 +328,14 @@ $body = @'
 
 Write-Console "Registering Fabric.Installer."
 $installerClientSecret = Add-ClientRegistration -authUrl $identityServerUrl -body $body
-Add-SecureInstallationSetting "common" "fabricInstallerSecret" $installerClientSecret $signingCert
+if($installerClientSecret){ Add-SecureInstallationSetting "common" "fabricInstallerSecret" $installerClientSecret $signingCert }
```

Comment:
Are the 2 lines below intentional (adding the same setting to "common" and "identity")?

Reply:
Yes, they are intentional - probably overkill but we allow and individual setting to override the common setting.